Assets/Scritps/ListExtension.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+
+public static class ListExtension
+{
+    public static void Shuffle<T>(this List<T> list, int shuffleTime)
```

Comment:
I think this would be nicer if it was an IEnumerable or an IList

Reply:
It will be IList, because the IEnumerable doesn't have braces